### PR TITLE
Fix Display issue

### PR DESCRIPTION
**Pull request by marqdevx**
* Display constructor fixed to last version updates, discover issue with `display.println()`
* Fix LEDs RGB order to `RGB` instead of `BGR`
* Bump version correctly (didnt do it in the latest release)

### CODE REVIEW

File: src/Arduino_MKRIoTCarrier.h [hunk 1]
```diff
@@ -125,6 +125,6 @@ class MKRIoTCarrier{
     Adafruit_ST7789 display = Adafruit_ST7789(TFT_CS, TFT_DC, -1);
 	
     //RGB LEDs
-    Adafruit_DotStar leds = Adafruit_DotStar(NUMPIXELS, DATAPIN, CLOCKPIN, DOTSTAR_BGR);
+    Adafruit_DotStar leds = Adafruit_DotStar(NUMPIXELS, DATAPIN, CLOCKPIN, DOTSTAR_RGB);
```
Comment (per1234) [inline src/Arduino_MKRIoTCarrier.h:128]: Is this correct?

I used this test sketch:

```cpp
#include <Arduino_MKRIoTCarrier.h>
MKRIoTCarrier carrier;
void setup() {
  carrier.begin();
  carrier.leds.setPixelColor(0, 255, 0, 0);
  carrier.leds.show();
}
void loop() {}
```

---

If I use the version of the library from the `master` branch, the LED on my MKR IoT Carrier turns red. That is the expected result from reading the referenced documentation:

https://adafruit.github.io/Adafruit_DotStar/html/class_adafruit___dot_star.html#ab7ffd239f3d7d1c3a3fa93449eac348e

>Parameters
n	Pixel index, starting from 0.
r	Red brightness, 0 = minimum (off), 255 = maximum.
g	Green brightness, 0 = minimum (off), 255 = maximum.
b	Blue brightness, 0 = minimum (off), 255 = maximum.

but not expected from reading this library's documentation:

https://github.com/arduino-libraries/Arduino_MKRIoTCarrier/blob/1d744f5262da09043ac33ee6235656f6d3b23b58/docs/readme.md#L223

---

When I use the library from this PR, the LED turns blue, which doesn't match either of them

**Review by per1234 — CHANGES_REQUESTED** [review]

File: src/Arduino_MKRIoTCarrier.h (hunk 1 above)
Comment (marqdevx) [reply src/Arduino_MKRIoTCarrier.h:128]: Need to check the order, for me was different, maybe I had some local changes 👀 
Fixed reference thought, thanks.

Comment (ubidefeo) [reply src/Arduino_MKRIoTCarrier.h:128]: @marqdevx @per1234 
it is incorrect

if you check my old PR you'll see that you need to specify `DOTSTAR_BGR` to have the correct order
https://github.com/arduino-libraries/Arduino_MKRIoTCarrier/pull/23/files

why have you changed it to RGB?

Comment (marqdevx) [reply src/Arduino_MKRIoTCarrier.h:128]: Tried and it the result was RGB when the dotstar was RGB, but then I had change the dotstar library probably, my fault. I'll revert the commit!

Comment (marqdevx) [reply src/Arduino_MKRIoTCarrier.h]: Dropped the commit, all should be good now @ubidefeo @per1234 😄

Comment (per1234) [reply src/Arduino_MKRIoTCarrier.h]: Thanks for updating the documentation @marqdevx.

**Pull request merged**